.github/workflows/e2e.yml

```diff
@@ -14,15 +14,29 @@ jobs:
       matrix:
         globalnet: ['', 'globalnet']
         lighthouse: ['', 'lighthouse']
+        ovn: ['', 'ovn']
+        exclude:
+          - ovn: 'ovn'
+            lighthouse: 'lighthouse'
+          - ovn: 'ovn'
+            globalnet: 'globalnet'
     steps:
       - name: Check out the repository
         uses: actions/checkout@v2
 
+      - name: Reclaim free space
+        run: |
+          # cleanup 30GB+ of tools we don't need for our CI
+          rm -rf /opt/ghc /usr/share/dotnet /usr/share/swift
+          df -h
+          free -h
+
```

Comment:
This shouldn’t be necessary, it’s taken care of by the shared e2e GHA (just below).

Reply:
> This shouldn’t be necessary, it’s taken care of by the shared e2e GHA (just below).

Done